Library/ViewModels/PledgeViewModel.swift

```diff
@@ -31,20 +41,50 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       .map { _ in AppEnvironment.current.currentUser }
       .map(isNotNil)
 
-    self.reloadWithData = Signal.combineLatest(project, reward, isLoggedIn)
+    let shippingAmount = Signal.merge(
+      self.shippingAmountSignal,
+      projectAndReward.mapConst(0)
```

Comment:
I'm assuming that the default shipping cost is coming from the shipping cell's view model once it's configured?

Reply:
Yes but not all rewards will have shipping so we map this to zero to ensure that this signal emits at least once.